scripts/apps/authoring/authoring/directives/AuthoringTopbarDirective.ts

```diff
@@ -61,6 +66,31 @@ export function AuthoringTopbarDirective(
             scope.isTranslationAvailable = function() {
                 return TranslationService.checkAvailability(scope.item);
             };
+
+            scope.extraActionsFromExtensions = [];
+
+            scope.triggerActionFromExtension = (actionToTrigger: IArticleAction) => {
+                actionToTrigger.onTrigger();
+            };
+
+            Promise.all(
+                Object.values(extensions)
+                    .map(({activationResult}) => {
+                        const getExtraActions = get(
```

Comment:
You are being too creative here :smile: Just copy-paste it from here and replace the path `https://github.com/superdesk/superdesk-client-core/blob/master/scripts/apps/search/components/actions-menu/MenuItems.tsx#L48`

- You don't need to map before using `flatMap`. You can return an empty array from `flatMap` callback and it will be ignored in the final result.
- Your current version uses `lodash.get` which means finding usages won't work.

Reply:
THe code was doing kind of the same, I'll update it to keep it consistent and avoid `get()`